Kiss.Bff.EndToEndTest/AnonymousContactmomentZaak/AnonymousContactmomentZaakScenarios.cs

```diff
@@ -0,0 +1,163 @@
+            await Step("Precondition: The scenario 'Search for Zaak in Contactmoment' has been executed");
+
+            await SearchForZaakInContactmoment();
+
+            await Step("Given the user is on the 'Zaak ZAAK-2023-001' screen");
+             
+            await Expect(Page.GetByRole(AriaRole.Heading, new() { Name = "Zaak ZAAK-2023-001" })).ToBeVisibleAsync();
+
+            await Step("When the user clicks Afronden in the Notes-Contactverzoek-Pane");
+
+            await Page.GetByRole(AriaRole.Button, new() { Name = "Afronden" }).ClickAsync();
+
+            await Step("And the Afhandeling form is displayed");
+
+            // This locator does not use an ARIA role. Consider updating the locator to use an ARIA role for better accessibility.
+            await Expect(Page.Locator("form.afhandeling")).ToBeVisibleAsync();
+
+            await Step("Then the heading 'Gerelateerde zaak' appears under 'Vraag 1'");
+
+            var vraagHeading = Page.GetByRole(AriaRole.Heading, new() { Name = "Vraag 1" });            
+
+            await Expect(vraagHeading.Locator("..").GetByRole(AriaRole.Heading, new() { Name = "Gerelateerde zaak" })).ToBeVisibleAsync();
+
+            await Step("And the item 'ZAAK-2023-001' is listed");
+
+            await Expect(Page.GetByText("ZAAK-2023-001")).ToBeVisibleAsync();
+
+            await Step("And a checked checkbox is visible");
+
+            await Expect(Page.GetByRole(AriaRole.Checkbox, new() { Name = "ZAAK-2023-001" })).ToBeCheckedAsync();
+        }
+
+        [TestMethod("3. Register Contactmoment bij Zaak - II")]
+        public async Task RegisterContactmomentBijZaakII()
+        {
+            await Step("Precondition: Scenario: Register Contactmoment bij Zaak – I is executed");
+
+            await RegisterContactmomentBijZaakI();
+
+            await Step("Given the user is on the Afhandeling-form");
+
+            // This locator does not use an ARIA role. Consider updating the locator to use an ARIA role for better accessibility.
+            await Expect(Page.Locator("form.afhandeling")).ToBeVisibleAsync();
+
+            await Step("When user enters value 'Contactmoment bij ZAAK-2023-001' in field Specifieke vraag");
+
+            await Page.GetByRole(AriaRole.Textbox, new() { Name = "Specifieke vraag" }).FillAsync("Contactmoment bij ZAAK-2023-001");
+
+            await Step("And user enters 'Live chat' in field Kanaal");
+
+            await Page.GetByLabel("Kanaal").SelectOptionAsync(new[] { new SelectOptionValue { Label = "Live Chat" } });
+
+            await Step("And value 'Zelfstandig afgehandeld' in field Afhandeling");
+
+            await Page.GetByRole(AriaRole.Combobox, new() { Name = "Afhandeling" }).SelectOptionAsync(new[] { "Zelfstandig afgehandeld" });
+
+            await Step("And selects value 'Parkeren' in field Afdeling");
+
+            // This locator does not use an ARIA role. Consider updating the locator to use an ARIA role for better accessibility.
```

Comment:
It does have an aria role, but no associated label. Can you make the comment a bit more clear?